addons/calendar/static/src/views/attendee_calendar/common/attendee_calendar_common_popover.js

```diff
@@ -67,6 +69,11 @@ export class AttendeeCalendarCommonPopover extends CalendarCommonPopover {
     get isEventArchivable() {
         return false;
     }
+    
+    async onClickAction(actionName) {
+        const action = await this.orm.call('calendar.event', actionName, [this.props.record.id]);
```

Comment:
One option would be to have `onClickAction` not receive any parameters, so in the XML you can have `t-on-click="onClickAction"` and inside the function you can access `this.fieldInfo.options.action`. Both options are ok for me, do as you prefer :slightly_smiling_face: